scripts/zones/Ilrusi_Atoll/instances/extermination.lua

```diff
@@ -1,69 +1,75 @@
 -----------------------------------
 -- Assault: Extermination
+-- Instance: 6602
 -----------------------------------
-require("scripts/globals/instance")
 local ID = require("scripts/zones/Ilrusi_Atoll/IDs")
+require("scripts/globals/assault")
+require("scripts/globals/instance")
 -----------------------------------
 local instanceObject = {}
 
-instanceObject.afterInstanceRegister = function(player)
-    local instance = player:getInstance()
+instanceObject.registryRequirements = function(player)
+    return player:hasKeyItem(xi.ki.ILRUSI_ASSAULT_ORDERS) and
+        player:getCurrentAssault() == xi.assault.mission.EXTERMINATION and
+        player:getCharVar("assaultEntered") == 0 and
+        player:hasKeyItem(xi.ki.ASSAULT_ARMBAND) and
+        player:getMainLvl() > 70
+end
 
-    player:messageSpecial(ID.text.ASSAULT_43_START, 43)
-    player:messageSpecial(ID.text.TIME_TO_COMPLETE, instance:getTimeLimit())
+instanceObject.entryRequirements = function(player)
+    return player:hasKeyItem(xi.ki.ILRUSI_ASSAULT_ORDERS) and
+        player:getCurrentAssault() == xi.assault.mission.EXTERMINATION and
+        player:getCharVar("assaultEntered") == 0 and
+        player:getMainLvl() > 70
 end
 
 instanceObject.onInstanceCreated = function(instance)
-    for i, v in pairs(ID.mob[43]) do
-        SpawnMob(v, instance)
-    end
-
-    GetNPCByID(ID.npc.RUNE_OF_RELEASE, instance):setPos(290.857, -3.424, 132.339, 148)
-    GetNPCByID(ID.npc.ANCIENT_LOCKBOX, instance):setPos(293.637, -3.376, 130.364, 148)
-    GetNPCByID(ID.npc._1jo, instance):setAnimation(8)
-    GetNPCByID(ID.npc._jj3, instance):setAnimation(8)
-    GetNPCByID(ID.npc._jj5, instance):setAnimation(8)
-    GetNPCByID(ID.npc._jjc, instance):setAnimation(8)
```

Comment:
Are these doors that are open in the sql tables now?

Reply:
Nope! Their animations are set to 8 by default in npc_list as they should